src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "WHITE";
+    public static final int BOUND_MAX = 10000;
+    private Random random = new Random();
+    private int randomRadius = random.nextInt(BOUND_MAX);

```

Comment:
All these "constants" are redundant, you could've replaced it with single "randomLength", but it's not a proper way to generate a "random" figure - let me explain this below.

Reply:
I think I don't understand. If I change for example BOUND_MAX into Integer.MAX_VALUE, then the Area values could be negative.

Reply:
What I meant was to create a single variable for random length value, or even better, call `nextInt` directly in the figure constructor call (for example `new Circle(random.nextInt(BOUND_MAX))`) instead of creating a random value for each dimension (randomRadius, randomSide, randomHeight) because we're creating a lot of redundant data.